### PR TITLE
SS4 upgrade.

### PR DESCRIPTION
**Pull request by mfendeksilverstripe**
Module updated to SilverStripe 4.

### CODE REVIEW

File: src/MimeUploadValidator.php [hunk 1]
```diff
@@ -59,13 +79,16 @@ public function isValidMime()
...
             }
         }
+
         return false;
     }
 
     /**
      * Fetches an array of valid mimetypes.
      *
+     * @param $tmpFile
```
Comment (robbieaverill) [inline src/MimeUploadValidator.php]: `@param array $tmpFile`

File: tests/MimeUploadValidatorTest.php [hunk 2]
```diff
@@ -1,4 +1,12 @@
 <?php
+
```
Comment (robbieaverill) [inline tests/MimeUploadValidatorTest.php]: Would you mind adding namespaces to the tests too? `SilverStripe\MimeValidator\Test` - add PSR-4 autoloader for this too to composer.json

File: composer.json [hunk 3]
```diff
@@ -12,13 +12,18 @@
...
 	"require": {
-		"silverstripe/framework": "~3.1"
+		"silverstripe/framework": "^4.0"
 	},
 	"extra": 
 	{
 		"branch-alias": 
 		{
-			"dev-master": "1.0.x-dev"
+			"dev-master": "2.0.x-dev"
```
Comment (robbieaverill) [inline composer.json]: I've updated this in master, can you rebase please

**Comment by mfendeksilverstripe** [comment]
@robbieaverill It's rebased now. Tests are also all green.

**Pull request merged**